src/components/AdditionalProjects/test.py

```
@@ -0,0 +1,210 @@
+page = 0
+retry_interval = 2
+tries = 4
+
+try:
+   while page < allpages:
+      try:
+        page = page +1
+        time.sleep(2)
+        print("going to " + str(page))
+        try:
+          for i in range(0, tries):
```

Ruff (0.8.0)

Loop control variable `i` not used within loop body

(B007)